.kitchen.yml

```diff
@@ -10,9 +10,22 @@ platforms:
   - name: ubuntu-12.04
   - name: centos-7.0
   - name: centos-6.5
+  <% if ENV['TEST_WINDOWS'] %>
```

Comment:
I do not see a reason why we need to keep the Windows test support on an environment variable? I've removed it locally and all looks good to me. Please remove this bit.